doc/source/building/index.rst

```diff
@@ -0,0 +1,364 @@
+
+  .. tab-item:: macOS
+
+    Install Apple Developer Tools. An easy way to do this is to
+    `open a terminal window <https://blog.teamtreehouse.com/introduction-to-the-mac-os-x-command-line>`_,
+    enter the command::
+
+        xcode-select --install
+
+    and follow the prompts. Apple Developer Tools includes Git, the Clang C/C++
+    compilers, and other development utilities that may be required.
+
+    Do *not* use the macOS system Python. Instead, install Python
+    with `the python.org installer <https://www.python.org/downloads/>`__ or
+    with a package manager like Homebrew, MacPorts or Fink.
+
+    The other system dependencies you need are a Fortran compiler, BLAS and
+    LAPACK libraries, and pkg-config. They're easiest to install with
+    `Homebrew <https://brew.sh/>`__::
+
+        brew install gfortran openblas pkg-config
+
+    .. note::
+
+        As of SciPy >=1.2.0, we do not support compiling against the system
+        Accelerate library for BLAS and LAPACK. It does not support a sufficiently
+        recent LAPACK interface. This is planned to change in 2023, because macOS
+        13.3 introduced a major upgrade to Accelerate which resolved all known
+        issues.
+
+  .. tab-item:: Windows
+
+    A compatible set of C, C++ and Fortran compilers is needed to build SciPy.
+    This is trickier on Windows than on other platforms, because MSVC does not
+    support Fortran, and gfortran and MSVC can't be used together. You will
+    need one of these sets of compilers:
+
+    1. Mingw-w64 compilers (``gcc``, ``g++``, ``gfortran``) - *recommended,
+       because it's easiest to install and is what we use for SciPy's own CI
+       and binaries*
+    2. MSVC + Intel Fortran (``ifort``)
+    3. Intel compilers (``icc``, ``ifort``)
+
+    Compared to macOS and Linux, building SciPy on Windows is a little more
+    difficult, due to the need to set up these compilers. It is not possible to
+    just call a one-liner on the command prompt as you would on other
+    platforms.
+
+    First, install Microsoft Visual Studio - the 2019 Community Edition or any
+    newer version will work (see the
+    `Visual Studio download site <https://visualstudio.microsoft.com/downloads/>`__).
+    This is needed even if you use the MinGW-w64 or Intel compilers, in order
+    to ensure you have the Windows Universal C Runtime (the other components of
+    Visual Studio are not needed when using Mingw-w64, and can be deselected if
+    desired, to save disk space).
+
+    .. tab-set::
+
+      .. tab-item:: MinGW-w64
+
+        There are several sources of binaries for MinGW-w64. We recommend the
+        RTools versions, which can be installed with Chocolatey (see
+        Chocolatey install instructions `here <https://chocolatey.org/install>`_)::
+
+            choco install rtools -y --no-progress --force --version=4.0.0.20220206
+            choco install pkgconfiglite  # not needed in a conda env
```

Comment:
I think pkg-config might be provided by rtools. At least it doesn't need an extra install step in the CI config.